spec/lib/karafka/connection/api_adapter_spec.rb

```diff
@@ -14,12 +14,12 @@
   end
 
   describe '#client' do
-    subject(:config) { described_class.client(consumer_group) }
+    subject(:config) { described_class.client }
 
     let(:expected_keys) { (attributes_map_values[:consumer] + %i[group_id]).sort }
 
-    it 'not to have any config_adapter keys' do
-      keys = config.last.keys - Karafka::AttributesMap.config_adapter.values.flatten
+    it 'not to have any api_adapter keys' do
+      keys = config.last.keys - Karafka::AttributesMap.api_adapter.values.flatten
```

Comment:
The config API doesn't look nice. I wouldn't expect it to be an Array. Is there a reason it looks like this?